Tests/Fixtures/TestProject/project.yml

```diff
@@ -113,12 +113,16 @@ targets:
       PRODUCT_BUNDLE_IDENTIFIER: com.project.app
     dependencies:
       - target: Framework_iOS
+        platform: mac+iOS
```

Comment:
```suggestion
        platform: all
```

Reply:
Updated in dfc471d8511f37738bfb9a84915d822bcc299f1e.